index.js

```diff
@@ -7,13 +7,6 @@ if (require.main === module) {
   var config = require('config');
   var node = BitcoreNode.create(config.get('BitcoreNode'));
   node.start();
-  node.on('error', function(err) {
-    if (err.code === 'ECONNREFUSED') {
-      console.log('Connection to bitcoind failed');
-    } else {
-      console.log('Unrecognized error: ', err);
-    }
-  });
```

Comment:
why did you remove this? I see you added a 'error' handler in BitcoreNode, but events are generally consumed outside of the `EventEmitter` object instance